modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/spaces/NewSpaceFormSection.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayForm from '@clayui/form';
+import ClayLayout from '@clayui/layout';
+import Link from '@clayui/link';
+import {sub} from 'frontend-js-web';
+import React, {PropsWithChildren} from 'react';
+
+import {PageLogo} from './PageLogo';
+
+interface NewSpaceFormSectionProps {
+	description: string;
+	linkLabel: string;
+	linkUrl: string;
+	onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+	step: 1 | 2;
+	title: string;
+}
+
+export function NewSpaceFormSection({
+	children,
+	description,
+	linkLabel,
+	linkUrl,
+	onSubmit,
+	step,
+	title,
+}: PropsWithChildren<NewSpaceFormSectionProps>) {
+	return (
+		<ClayLayout.Col className="mw-50 px-9 w-50">
```

Comment:
I would prefer to move this <ClayLayout.Col> to the parent, so in NewSpace it is clear that there are two columns, and in NewSpaceFormSection we only have the form